src/styles/_utility.scss

```diff
@@ -178,7 +179,7 @@ $scales-map: (
       2: 3.8vh,
       3: 4vh,
       4: 4.3vh,
-      5: 4.5vh,
+      5: 4.4vh,
```

Comment:
Ditto, this should be reflected in `$line-heights-map` and `$skeleton-height-map`.